src/commands/GameIntegration/clashofclans.ts

```diff
@@ -0,0 +1,156 @@
+import { toTitleCase } from '@klasa/utils';
+import { MessageEmbed } from 'discord.js';
+import { CommandStore, KlasaMessage } from 'klasa';
+import { TOKENS } from '../../../config';
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { UserRichDisplay } from '../../lib/structures/UserRichDisplay';
+import { ClashOfClans } from '../../lib/types/definitions/ClashOfClans';
+import { fetch, FetchResultTypes, getColor } from '../../lib/util/util';
+import { BrandingColors } from '../../lib/util/constants';
+
+const enum ClashOfClansFetchCategories {
+	'players' = 'players',
+	'clans' = 'clans'
+}
+
+export default class extends SkyraCommand {
+
+	private PLAYER_TAG_REGEX = /#[A-Z0-9]{3,}/;
```

Comment:
And update all references:
```suggestion
	private readonly kPlayerTagRegex = /#[A-Z0-9]{3,}/;
```